src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/endringer/UtledEndringerUtil.kt

```diff
@@ -52,10 +52,9 @@ object UtledEndringerUtil {
         formatterEndring(BarnDto::borHosSøker, "Bor hos søker"),
         formatterEndring(BarnDto::dødsdato, "Dødsdato"),
         formatterEndring(BarnDto::fødselsdato, "Fødselsdato"),
-        formatterEndring({ it.annenForelder?.personIdent }, "Annen forelder")
-        // Todo Mattis - ta med disse i neste PR (delt bosted i vilkår)
-        // formatterEndring(BarnDto::harDeltBostedNå, "Har delt bosted nå"),
-        // formatterEndring(BarnDto::deltBosted, "Delt bosted"),
+        formatterEndring({ it.annenForelder?.personIdent }, "Annen forelder"),
+        formatterEndring(BarnDto::harDeltBostedNå, "Har delt bosted nå"),
+        formatterEndring(BarnDto::deltBosted, "Delt bosted")
```

Comment:
Trenger vi begge disse? Kan nog være fint å høre med funksjonelle og forklare de tilfeller som kan oppstå og høre hva som er interessant å vise

Reply:
Kanskje det holder med delt bosted ?

Reply:
harDeltBostedNå - er vel ikke relevant i forhold til å sjekke om grunnlagsdata har endret seg? 
Kanskje det må en litt større endring til for å få dette til å fly? 

Mulig vi kunne splittet i to PR'er - "vis fram delt bosted" i vilkår og "vis fram endring i grunnlagsdata på delt bosted"?

Reply:
Hørte med fag - joda, vi må sjekke begge deler. Fint å si ifra om at status "nå" er annerledes enn status når grunnlagsdata ble opprettet har endret seg. (Her har jo ikke nødvendigvis grunnlagsdata endret seg, bare tiden).